### PR TITLE
Can 595

### PR DESCRIPTION
**Pull request by shahab-ramzan**
Popup messages updated

### CODE REVIEW

File: src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx [hunk 1]
```diff
@@ -127,7 +127,9 @@ const NewsFeedsCard = ({ newsFeed }) => {
                       )}
                       {deleteNews && (
                         <Tooltip
-                          title={news.owner_flag ? "" : "must owner to delete"}
+                          title={
+                            news.owner_flag ? "" : " must be owner to delete"
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx:131]: Message must comes from the common file

File: src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx [hunk 2]
```diff
@@ -151,7 +153,7 @@ const NewsFeedsCard = ({ newsFeed }) => {
 
                       {editNews && (
                         <Tooltip
-                          title={news.owner_flag ? "" : "must owner to edit"}
+                          title={news.owner_flag ? "" : "must be owner to edit"}
```
Comment (akuks) [inline src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx]: Message must must come from the common file

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx:131]: Done

File: src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx (hunk 2 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx]: Done

**Pull request merged**